src/Helpers/Portfolio/PortfolioHelper.js

```diff
@@ -59,3 +59,11 @@ export async function updatePortfolio(portfolioData) {
     window.console.error(error);
   });
 }
+
+export async function removePortfolio(portfolioId) {
+  return userApi.destroy_portfolio(JSON.stringify ({ portfolio_id: portfolioId })).then(() => {
+    console.log('Remove Portfolio Called successfully.');
+  }, (error) => {
```

Comment:
This is not a good way to handle exceptions. If you don't throw Error here, the promise handling from insights notifications will not run and the app will crash in runtime instead of creating notification with errors.